src/core/saca-hub/saca-hub.front.json

```diff
@@ -177,6 +177,85 @@
 				"default_val": "mlzDemoBastionHostIpCfg",
 				"description": "The name of the Bastion Host IP Configuration",
 				"options": []
+			},
+			{
+				"varname": "jumpbox_subnet.name",
+				"type": "text",
+				"default_val": "mlzDemoJumpboxSubnet",
+				"description": "The name of the jumpbox subnet",
+				"options": []
+			},
+			{
+				"varname": "jumpbox_subnet.address_prefixes",
+				"type": "list",
+				"default_val": [
+					"10.0.100.160/27"
+				],
+				"description": "The address space(s) (new line seperated) to be used for the jumpbox subnet.",
+				"options": []
+			},
+			{
+				"varname": "jumpbox_subnet.nsg_name",
+				"type": "text",
+				"default_val": "mlzDemoJumpboxSubnetNsg",
+				"description": "The name of the jumpbox subnet route table network security group",
+				"options": []
+			},
+			{
+				"varname": "jumpbox_subnet.routetable_name",
+				"type": "text",
+				"default_val": "mlzDemoJumpboxSubnetRt",
+				"description": "The name of the jumpbox subnet route table",
+				"options": []
+			},
+			{
+				"varname": "jumpbox_keyvault_name",
+				"type": "text",
+				"default_val": "mlzDemoJumpboxVmKv",
```

Comment:
The error I got when deploying could also be related to this default.

Reply:
Will generate a name